packages/daemon/daemon-docs.md

```diff
@@ -0,0 +1,112 @@
+# `@endo/cli`
+
+### `endo start`
+
+- executes [stop](#start) function definined within the `@endo/daemon`.
+
+### `endo stop`
+
+- executes [stop](#stop) function definined within the `@endo/daemon`.
+
+### `endo clean`
+
+- executes the [`clean`](#clean) function definined within the `@endo/daemon`.
```

Comment:
To clarify - "resetting to factory defaults" is to reset all settings *that haven't been saved*? I'm inferring that it must be or otherwise this metaphor would apply to `endo reset`.

Reply:
I was mistaken. `clean` only deletes the domain socket or named pipe.

I expect that we’ll just get rid of this command. I used it while debugging.